ProjectsJenkinsfile

```diff
@@ -84,19 +84,48 @@ def dub_projects = [
    "msoucy/dproto",
    "nomad-software/dunit",
    "rejectedsoftware/diet-ng",
-   "rejectedsoftware/vibe.d",
+   "rejectedsoftware/vibe.d#libevent",
+   "rejectedsoftware/vibe.d#libasync",
    "repeatedly/mustache-d",
    "s-ludwig/taggedalgebraic",
 ]
 
 def runTest(name) {
     def n = name // to fix issues with closure
+    def repo = name.split('#')[0]
+    echo repo
```

Comment:
No it's separate checkouts.
Vibe.d is the longest running test in the suite, but libasync contributes only a small part.
Let's remove that and maybe parallize vibe.d's test suite at some point instead.